spec/controllers/api/v1/computer_accessories_controller_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ComputerAccessoriesController, type: :controller do
+
+  let(:user) { create(:user_with_token, :with_can_read_or_modify_user_permission, role_name: 'Reviewer') }
+  let(:computer_accessory_params) { FactoryBot.attributes_for(:computer_accessory) }
+
+  before do
+    generate_and_set_token(user)
+    @record = build(:computer_accessory)
+    allow(Stockit::ItemDetailSync).to receive(:create).with(@record).and_return({"status"=>201, "computer_accessory_id"=> 12})
+    @record.save
+
+    serialized_record_with_country = Api::V1::ComputerAccessorySerializer.new(@record, include_country: true).as_json
+    @parsed_body_with_country = JSON.parse( serialized_record_with_country.to_json )
+
+    serialized_record_without_country = Api::V1::ComputerAccessorySerializer.new(@record, include_country: false).as_json
+    @parsed_body_without_country = JSON.parse(serialized_record_without_country.to_json)
+  end
+
+  describe "GET computer_accessories" do
+    it "returns 200", :show_in_doc do
```

Comment:
Please add updated `apipie_examples.json` as here `: show_in_doc` is used.
https://github.com/crossroads/api.goodcity/blob/master/doc/apipie_examples.json